core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -185,6 +251,31 @@ var AdminPage = function() {
     await this.saveChangeOfFeature(featureElement);
   };
 
+  this.enableFeatureForDev = async function(featureElement) {
+    await this.removeAllRulesOfFeature(featureElement);
+
+    await action.click(
+      'Add feature rule button',
+      featureElement
+        .$(addFeatureRuleButtonLocator)
+    );
+
+    await waitFor.visibilityOf(
+      featureElement.$(valueSelectorLocator),
+      'Value Selector takes too long to appear'
+    );
```

Comment:
Why we are checking for the visiblity of here?

Reply:
Previously we were clicking on the option itself to select, and we were using `action.click()` for that, which actually waits for the element to be visible first. So here also i am firstly waiting for the element to be visible before we can call `selectByVisibleText` on it.